manifests/klusterlet/managed/klusterlet-registration-clusterrole.yaml

```diff
@@ -22,7 +22,7 @@ rules:
 - apiGroups: ["cluster.open-cluster-management.io"]
   resources: ["clusterclaims"]
   verbs: ["get", "list", "watch"]
-# Allow agent to list addons lease
+# Allow agent to operate leases
 - apiGroups: ["coordination.k8s.io"]
   resources: ["leases"]
-  verbs: ["get", "list", "watch"]
+  verbs: ["create", "get", "list", "watch", "update", "patch"]
```

Comment:
these are agent rbac on hub I think, lease is not needed

Reply:
this is the agent clusterrole on spoke, we already add the lease permission to the spoke agent role (`klusterlet/management/klusterlet-registration-role.yaml`) , so rollbacked